commit-patch-buffer.el

```diff
@@ -152,7 +152,7 @@ following algorithm:
     (beginning-of-buffer)
     (diff-hunk-next) ;; Have to be in a hunk or diff-hunk-file-names won't work.
     (let ((diff-path (reverse (split-string (car (diff-hunk-file-names)) "/")))
-          (file-path (reverse (split-string (buffer-file-name (car (diff-find-source-location))) "/"))))
+          (file-path (reverse (split-string (file-chase-links (buffer-file-name (car (diff-find-source-location)))) "/"))))
```

Comment:
Don't we need to have this on the result of `(car (diff-hunk-file-names))` as well?